source/server/getting-started/implementations/spongeforge.rst

```diff
@@ -42,8 +42,8 @@ Example
 
 SpongeForge Jar files will always follow this naming scheme, to allow you to easily identify compatibility.
 
-For example the file name ``spongeforge-1.10.2-2254-5.2.0-BETA-2234.jar`` is compatible with Minecraft version
-``1.10.2``, was built with Forge ``12.18.3.2254`` (Build ``2254``), provides SpongeAPI ``5.2.0`` and was build number
+For example the file name ``spongeforge-1.12.2-2611-7.1.0-BETA-2990.jar`` is compatible with Minecraft version
+``1.12.2``, was built with Forge ``114.23.2.2611`` (Build ``2990``), provides SpongeAPI ``7.1`` and was build number
 ``2234`` of SpongeForge.
```

Comment:
`was built with Forge ``114.23.2.2611`` (Build ``2611``), provides SpongeAPI ``7.1.0`` and was build number
 ``2290`` of SpongeForge.`

Reply:
That should be ``2990`` not ``2290`` ...

Reply:
numbers are hard!